### DISCUSSION TITLE
Loading gwyddion file contribution

### DISCUSSION THREAD

**Discussion by iobataya** [AFM-SPM/TopoStats#771] [Show and tell]
Hi,

Now I finished coding and testing for the known issues as discussed in pull request #659.

[New branch in my repository: gwy_load_arb_channel_num_and_flipped_xy](https://github.com/AFM-SPM/TopoStats/compare/main...iobataya:TopoStats:PR659-gwy_load_arb_channel_num_and_flipped_xy)

We've discussed about loading gwyddion file before in another pull request #659.  Since I failed to manage pull the latest version of Topostats without conflicts in my forked repository, I re-forked the original repository. By the re-forking, the pull request was closed automatically. So I made a new branch in the re-forked repository.  I'm not sure if this category is appropriate or not, but I thought it right to show and tell here.

Though pre-commit still complained 'too complex', tests are passed and coverage of test_io.py was 84%.  I think I can try pull request now. If I miss something to check in advance, I'd appreciate your advice.

Thanks,
Ikuo

**Comment by ns-rse** [comment]
Hi @iobataya ,

That sounds great and thanks for finding the time to come back to this.

I look forward to your PR.

@ns-rse
